webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js

```diff
@@ -0,0 +1,224 @@
+        dispatch(removeCVFilterRule(filterId, packagefilterId, () =>
+          dispatch(getContentViewDetails(cvId))));
+      },
+    },
+  ];
+
+  const emptyContentTitle = __('No rules have been added to this filter.');
+  const emptyContentBody = __("Add to this filter using the 'Add filter rule' button.");
+  const emptySearchTitle = __('No matching filter rules found.');
+  const emptySearchBody = __('Try changing your search settings.');
+
+
+  return (
+    <Tabs activeKey={activeTabKey} onSelect={(_event, eventKey) => setActiveTabKey(eventKey)}>
+      <Tab eventKey={0} title={<TabTitleText>{__('Module Streams')}</TabTitleText>}>
+        <div className="tab-body-with-spacing">
+          <TableWrapper
+            {...{
+              rows,
+              metadata,
+              emptyContentTitle,
+              emptyContentBody,
+              emptySearchTitle,
+              emptySearchBody,
+              searchQuery,
+              updateSearchQuery,
+              error,
+              status,
+              actionResolver,
+            }}
+            status={status}
+            onSelect={onSelect(rows, setRows)}
+            cells={columnHeaders}
+            variant={TableVariant.compact}
+            autocompleteEndpoint={`/module_streams/auto_complete_search?filterid=${filterId}`}
+            fetchItems={useCallback(params =>
+              getCVFilterModuleStreams(cvId, filterId, params), [cvId, filterId])}
+            actionButtons={
+              <Split hasGutter>
+                <SplitItem>
+                  <Button isDisabled={!hasNotAddedSelected} onClick={bulkAdd} variant="secondary" aria-label="add_filter_rule">
+                    {__('Add filter rule')}
+                  </Button>
+                </SplitItem>
+                <SplitItem>
+                  <Dropdown
+                    toggle={<KebabToggle aria-label="bulk_actions" onToggle={toggleBulkAction} />}
+                    isOpen={bulkActionOpen}
+                    isPlain
+                    dropdownItems={[
+                      <DropdownItem aria-label="bulk_add" key="bulk_add" isDisabled={!hasNotAddedSelected} component="button" onClick={bulkAdd}>
+                        {__('Add')}
+                      </DropdownItem>,
+                      <DropdownItem aria-label="bulk_remove" key="bulk_remove" isDisabled={!hasAddedSelected} component="button" onClick={bulkRemove}>
+                        {__('Remove')}
+                      </DropdownItem>]
+                  }
+                  />
+                </SplitItem>
+              </Split>
+            }
+          />
+        </div>
+      </Tab>
+      {(repositories.length || showAffectedRepos) &&
+      <Tab eventKey={1} title={<TabTitleText>{__('Affected Repositories')}</TabTitleText>}>
+        <div className="tab-body-with-spacing">
+          <AffectedRepositoryTable cvId={cvId} filterId={filterId} repoType="yum" setShowAffectedRepos={setShowAffectedRepos} />
```

Comment:
We need this effect for the functionality where selecting to apply to a subset of repos takes user to the tab etc..You'll find this in other filter pages..We might need to refactor this somehow but for now, adding this is needed.
```
useEffect(() => {
    if (!repositories.length && showAffectedRepos) {
      setActiveTabKey(1);
    } else {
      setActiveTabKey(0);
    }
  }, [showAffectedRepos, repositories.length]);
```